src/main/java/seedu/address/logic/commands/EditCustomerCommand.java

```diff
@@ -104,7 +105,13 @@ private static Customer createEditedCustomer(Customer customerToEdit,
         Set<Tag> updatedTags = editCustomerDescriptor.getTags().orElse(customerToEdit.getTags());
         CustomerId id = customerToEdit.getId();
 
-        return new Customer(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags, id);
+        Customer retCustomer;
+        try {
+            retCustomer = new Customer(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags, id);
+        } catch (IllegalArgumentException e) {
+            throw new CommandException(e.getMessage());
```

Comment:
This should be fine but what model validation logic does Customer have?

Reply:
Shouldn't be necessary unless this is for future cases and consistency across the models.